lib/ModelMixins/TableMixin.ts

```diff
@@ -48,6 +49,24 @@ function TableMixin<T extends Constructor<Model<TableTraits>>>(Base: T) {
       ChartableMixin(DiscretelyTimeVaryingMixin(CatalogMemberMixin(Base)))
     )
     implements SelectableDimensions {
+    /**
+     * The default {@link TableStyle}, which is used for styling
+     * only when there are no styles defined.
+     */
+    private readonly defaultTableStyle: TableStyle;
+
+    constructor(...args: any[]) {
+      super(...args);
+
+      const tableStyle = new TableStyle(this, -1);
```

Comment:
what's a style -1? Can we make the stylenumber an optional parameter in TableStyle so that if a styleNumber isn't provided, it falls back to a default style?

Reply:
Done!